build/ci/official.yml

```diff
@@ -129,11 +129,21 @@ steps:
     ArtifactType: Container
   condition: succeededOrFailed()
 
+- task: NuGetToolInstaller@0
+  inputs:
+  versionSpec: '4.9.2'
+
+# Authenticate with service connections to be able to publish packages to external nuget feeds.
+- task: NuGetAuthenticate@0
+  inputs:
+  nuGetServiceConnections: azure-public/vs-impl, azure-public/vssdk
```

Comment:
Why do we need VSSDK as a connection if we're publishing to vs-impl?

Reply:
This one is still a draft, just copied over the same thing Roslyn is doing - don't actually think we even need this step if the NuGet Publisher task succeeds. They need to Authenticate because they're using a ps1